docs/guides/databases_with_concord.md

```diff
@@ -2,14 +2,134 @@
 
 Sometimes, when you are developing a bot command, you may wish to save some data that will not be erased once the bot restart (aka persistent data). This is where a database comes in - it can save data, and don't worry, it won't be erased if you restart your bot.
 
-In this guide, you'll be learning how to use sqlite3, a local database. It's incredibly fast, easy to use, and simple to debug.
+In this guide, you will either decide on choosing SQLite3 to start, or postgreSQL, which is the most recommended one for production.
```

Comment:
In this guide, you will need to using either SQLite3 or PostgreSQL. Postgres is strongly recommended for production bots, as Sqlite3 is not as performant.

Reply:
Oh yes, I only mean it was fast, but I think your changes made it better, thanks!